src/software/jetson_nano/display/screens/home_screen.py

```diff
@@ -21,15 +21,15 @@ def __init__(self, lcd_display, redis_dict, screen_actions):
         """
         actions = [
             {
-                "redis key": "robot id",
-                "value": redis_dict["robot id"],
+                "redis key": "/robot_id",
```

Comment:
The constants should also be used in this file as well.

Reply:
seems like a decent difficulty 1 ticket. tagged a ticket